src/schema/me/followed_fairs.ts

```diff
@@ -0,0 +1,56 @@
+import { ProfileType } from "schema/profile"
+import Fair from "schema/fair"
+import { IDFields } from "schema/object_identification"
+
+import { pageable, getPagingParameters } from "relay-cursor-paging"
+import { connectionDefinitions, connectionFromArraySlice } from "graphql-relay"
+import { GraphQLObjectType } from "graphql"
+
+const FollowedProfileEdge = new GraphQLObjectType({
```

Comment:
Rename this to remove the notion of “profile”, seeing as we’re modeling a direct relationship from user to fairs.